students/JeroenSweerts/Lesson10/mailroom6.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+from functools import reduce
+
+class Donor():
+    def __init__(self, name=None):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        """Property for the donor's name."""
+        return self._name
+
+    @name.setter
+    def name(self, val):
+        """Setter for the donor's name."""
+        self._name = val
+
+    @property
+    def donations(self):
+        """Property for the donor's list of donations."""
+        return self._donations
+
+    @donations.setter
+    def donations(self, val):
+        """Setter for the donor's list of donations."""
+        self._donations = val
+
+
+class Donor_Collection(dict):
+    def __init__(self, donors):
+        self._donors = donors
+
+    @property
+    def donors(self):
+        """Property for the Donor_Collection's list of donors."""
+        return self._donors
+
+    @donors.setter
+    def donors(self, val):
+        """Setter for the Donor_Collection's list of donors."""
+        self._donors = val
```

Comment:
there is no reason to use property on `donors`, setters and getters are useful if you need special logic on either getting value or setting value. you can just use `donors` directly